com.sap.cloud.lm.sl.cf.core/src/test/resources/com/sap/cloud/lm/sl/cf/core/cf/v2_0/mtad-10.yaml

```diff
@@ -0,0 +1,43 @@
+_schema-version: "2.0.0"
+ID: com.sap.sample.mta
+version: 1.0.0
+
+modules:
+  - name: module-1
+    type: javascript.nodejs
+    requires:
+      - name: service-key-1
+      - name: service-key-2
+      - name: service-key-3
+        parameters:
+          env-var-name: env-defined-in-required-dependency
+      - name: service-key-4
+        parameters:
+          env-var-name: env-defined-in-required-dependency-and-override
+
+resources:
+  - name: service-key-1
+    type: org.cloudfoundry.existing-service-key
+    parameters:
+      service-name: service-1
+      type: existing-service-key
```

Comment:
Why are you adding here the type as parameter? This should happen automatically by the descriptor processing.

Reply:
It is because the type of resources is defined in manifest in "com.sap.cloud.lm.sl.cf.web" and here in "com.sap.cloud.lm.sl.cf.core" we do not know the type of the resource.